#!/usr/bin/env python
PACKAGE = "dynamic_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#Sample line
#gen.add("neighbour_radius",	double_t,	0, "Filter param",	10,	10,	10)

#Filter params
gen.add("cutoff_min_depth",	double_t,	0, "Filter param",	0.25,	1.5,	20.0)
gen.add("cutoff_max_depth",	double_t,	0, "Skipped if == 0.25",	0.25,	5.0,	20.0)
gen.add("down_sample_radius",	double_t,	0, "Skipped if == 0",	0.0,	0.025,	0.50)
gen.add("neighbour_radius",	double_t,	0, "Filter param",	0.025,	0.25,	1.0)
gen.add("neighbour_radius",	double_t,	0, "Skipped if == 0",	0,	5,	100)

#Variance params


exit(gen.generate(PACKAGE, "rex_gridmapping", "dyn_rex_gridmapping"))